doc/source/data_structures.rst

```diff
@@ -55,39 +55,39 @@ in :doc:`geometric manipulations <geometric_manipulations>`.
 
 Attributes
 ^^^^^^^^^^^^^^^
-* ``area``: shape area (units of projection -- see :doc:`projections <projections>`)
-* ``bounds``: tuple of max and min coordinates on each axis for each shape
-* ``total_bounds``: tuple of max and min coordinates on each axis for entire GeoSeries
-* ``geom_type``: type of geometry.
-* ``is_valid``: tests if coordinates make a shape that is reasonable geometric shape (`according to this <http://www.opengeospatial.org/standards/sfa>`_).
+* :attr:`~geopandas.GeoSeries.area`: shape area (units of projection -- see :doc:`projections <projections>`)
```

Comment:
BTW, you can still remove the `geopandas.` part. If you don't want the GeoSeries to show up, only the `~` is needed.